spacy/errors.py

```diff
@@ -229,8 +229,9 @@ class Errors(metaclass=ErrorsWithCodes):
             "If you're using a Transformer, make sure to install 'spacy-transformers'. "
```

Comment:
I think this may now be more confusing than helpful.

```suggestion
```

Reply:
Ah, true. Overlooked that. Are we fine with suggesting both `spacy-transformers` and `spacy-curated-transformers` then? The other option would be to display the former for <=3.6.x and the latter for the rest.

Reply:
I think we'd just change it for the current v3.7 release, so this wouldn't affect v3.6. Overall it's getting too hairy (the error message is already extremely long) and I think it's better in general to assume that model packages handle their own dependencies. I think suggesting that people install both is not a good idea.

The main edge cases are mainly related to local saved files/directories, where you have a config or a local model directory that was generated in a venv with `spacy-transformers` (or whichever package) that you're now trying to load in a different venv. The package records the dependencies but the local files/dirs don't.

I guess I wouldn't be opposed to adding more targeted info to the error message if we can see from the missing registered function name that we should probably suggest package X (but we still can't do it 100% correctly because of the lack of actual namespacing).

Reply:
I still think we should take this line out now.

Reply:
Removed that line.